pkg/asset/agent/mirror/cabundle.go

```diff
@@ -53,15 +53,17 @@ func (i *CaBundle) Generate(_ context.Context, dependencies asset.Parents) error
 
 	switch agentWorkflow.Workflow {
 	case workflow.AgentWorkflowTypeInstall:
+		// once AgentWorkflowTypeInstallInteractiveDisconnected is reomved, the default workflow
+		// will be AgentWorkflowTypeInstall and then the cabundle asset should just return empty
+		// as its not needed.
+		// The thought here is in OVE, install config won't be supplied 
+		// ( but could cause issues when UI is introduced for regular cases)
```

Comment:
I don't expect it to cause issues in regular cases.
If there's a special case for additionalTrustBundle in the OVE installer (I'm not sure that there is), it's because of the internal registry and not because of the UI. We control the appliance builder so we can make sure it does the Right Thing here - the `unconfigured-ignition` command is not user-facing.